azure/app/template.json

```diff
@@ -183,6 +205,59 @@
         }
       }
     },
+
+    {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2017-05-10",
+      "name": "[variables('vspAppServicePlanDeploymentName')]",
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[concat(variables('deploymentUrlBase'), 'app-service-plan.json')]",
+          "contentVersion": "1.0.0.0"
+        },
+        "parameters": {
+          "appServicePlanName": {
+            "value": "[variables('vspAppServicePlanName')]"
+          }
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "apiVersion": "2016-08-01",
+      "name": "[variables('vspAppServiceName')]",
+      "kind": "app,linux,container",
+      "location": "[resourceGroup().location]",
+      "dependsOn": ["[variables('vspAppServicePlanDeploymentName')]"],
+      "properties": {
+        "httpsOnly": true,
+        "serverFarmId": "[variables('vspAppServicePlanId')]",
+        "siteConfig": {
+          "alwaysOn": "[parameters('appServiceAlwaysOn')]",
+          "httpLoggingEnabled": true,
+          "linuxFxVersion": "[variables('vspAppServiceRuntimeStack')]",
+          "appSettings": [
+            {
+              "name": "VERIFY_ENVIRONMENT",
+              "value": "[parameters('VSP.VERIFY_ENVIRONMENT')]"
+            },
+            {
+              "name": "SERVICE_ENTITY_IDS",
+              "value": "[string(createArray(parameters('appServiceHostName')))]"
```

Comment:
It's worth pointing out that this means that the `tmp` resource groups won't actually work with the VSP because they share `appServiceHostName` with their non-`tmp` pair. Their utility is mostly in testing the infrastructure deployments, but if you want to check it actually works, you'll probably need to deploy to a real environment.